pinecone/models/pod_spec.py

```diff
@@ -60,6 +60,11 @@ class PodSpec(NamedTuple):
     {'indexed': ['field1', 'field2']}
     """
 
+    source_collection: Optional[str] = None
+    """
+    The name of the collection to use as the source for the pod index. This configuration is only used when creating a pod index from an existing collection.
+    """
+
```

Comment:
This missing property was the cause create index with `source_collection` was failing.